src/styles/partials/components/_site-header.scss

```diff
@@ -39,20 +39,34 @@
 }
 
 @media screen and (max-width: $header-breakpoint) {
+  #search-form {
+    display: none;
+    padding: 15px;
+    position: absolute;
+    top: 85px;
+    height: 65px;
+    width: 100%;
+    background: white;
+    z-index: 999999999;
```

Comment:
same as above. Should try to use variables for values here if possible

Reply:
jsut relaized these shouldn't be showing up in this pr.